src/vellum/workflows/descriptors/tests/test_utils.py

```diff
@@ -77,6 +77,10 @@ class Outputs(BaseNode.Outputs):
         (FixtureState.zeta["foo"], "bar"),
         (ConstantValueReference(1), 1),
         (FixtureState.theta[0], "baz"),
+        (FixtureState.gamma.matches_regex(r"^h.*o$"), True),  # "hello" matches the regex
+        (FixtureState.gamma.matches_regex(r"^e.*o$"), False),  # "hello" does not match the regex
+        (FixtureState.delta.matches_regex(r"^el$"), True),  # "el" matches the regex exactly
+        (FixtureState.delta.matches_regex(r"^E.*"), False),  # Case-sensitive mismatch
```

Comment:
[nit] We should only need one of these four tests. They collectively are testing `re.match`, and are not testing different aspects of the SDK's implementation.